packages/graphback-datasync/src/services/createDataSyncService.ts

```diff
@@ -15,7 +15,7 @@ export interface CreateDataSyncCRUDServiceOptions {
  *
  * @param config
  */
-export function createDataSyncCRUDService(config: CreateDataSyncCRUDServiceOptions): (...args: any[]) => GraphbackCRUDService {
```

Comment:
So is it a non-typed function now?

Reply:
Proper types are inferred - specifying return types became antipattern